scripts/generate.sh

```diff
@@ -32,6 +32,7 @@ function install_did_files() {
       canister_name="${array[0]}"
       source="${array[1]}"
       filename="${source##*/}"
+      filename="${filename//-/_}"
```

Comment:
The cycles-ledger has a hyphen in the did file name.  dfx uses underscores instead.

Reply:
Can you add this comment in the code?

Reply:
Done